RetakesPlugin.cs

```diff
@@ -195,37 +195,43 @@ public void OnCommandShowSpawns(CCSPlayerController? player, CommandInfo command
         }
         
         // Pre cache the sprites.
-        Server.PrecacheModel("sprites/laserbeam.vmt");
+        // Server.PrecacheModel("sprites/laserbeam.vmt");
         
         foreach (var spawn in spawns)
         {
```

Comment:
Can you move the logic for showing a spawn into a `DisplaySpawn(Spawn spawn)...` helper and use that here instead?